samples/subsys/usb/mass/README.rst

```diff
@@ -122,6 +122,26 @@ follows:
    :goals: build
    :compact:
 
+For boards with TFM support a Non-Secure image can be build:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/mass
+   :board: stm32l562e_dk_ns
+   :goals: build
+   :compact:
+
+.. note::
+   Note that ``arm-none-eabi-gcc`` should be available in the PATH variable
+   and that ``STM32_Programmer_CLI`` is required to run ``regression.sh``
+   (see https://www.st.com/en/development-tools/stm32cubeprog.html). If you
+   are still having trouble running these scripts, check the Programming and
+   Debugging section of the :ref:`stm32l562e_dk_board` documentation.
+
+.. code-block:: console
+
+   ./build/tfm/regression.sh
+   west flash
```

Comment:
Honestly, I want to keep all samples generic, that is not quite the case for this one, but this changes going too far in my opinion, too board and configuration specific, sorry.

Reply:
I think this should just be cleaned up as this is not sample specific and is already documented at board level.